src/main/java/com/puppycrawl/tools/checkstyle/checks/UncommentedMainCheck.java

```diff
@@ -135,6 +138,14 @@
      */
     public static final String MSG_KEY = "uncommented.main";
 
+    /** Set of possible String array types. */
+    private static final Set<String> STRING_PARAMETER_NAMES = Stream.of(
```

Comment:
OOC (it has been a while since I've written much Java) could this be Set.of()?

Reply:
Not until Java 9 :)